en/docs/integrate/develop/creating-artifacts/data-services/securing-data-services.md

```diff
@@ -100,6 +101,8 @@ See the instructions [deploying the artifacts]({{base_path}}/integrate/develop/d
 
 Create a Soap UI project with the relevant security settings and then send the request to the hosted service.
 
+For guidelines on using SoapUI, see [general guidelines on testing with SOAP UI]({{base_path}}/integrate/develop/advanced-development/applying-security-to-a-proxy-service/#general-guidelines-on-testing-with-soap-ui).
+
```

Comment:
@arunans23 ,
Line113 has the following content
"See the instructions on [encrypting plain-text passwords](../../../setup/security/encrypting_plain_text)"
This link has to be fixed